mod/05_email/set/type/email_template.rb

```diff
@@ -5,7 +6,7 @@ def clean_html?
 def deliver args={}
   begin
     mail = format.render_mail(args)
-    mail.deliver 
+    mail.deliver
```

Comment:
without the change below, the "mail" variable can end up as a String and will fail here.